invenio_app_rdm/theme/templates/invenio_app_rdm/deposits/forms/deposits.html

```diff
@@ -13,8 +13,8 @@
 {%- endblock head_links %}
 
 {%- block css %}
-    {{ webpack['theme.css']}}
-    {{ webpack['invenio-app-rdm-search-theme.css']}}
+    {{ webpack['theme.css'] }}
+    {{ webpack['invenio-app-rdm-theme.css'] }}
```

Comment:
moderate: I think we can remove the full css block overwrite here, since this template extendes `BASE_TEMPLATE` which in RDM is `page.html`. This file already contain said css files (See [here]( https://github.com/inveniosoftware/invenio-app-rdm/blob/5f2bcf8cf54260d82ebdd6d0a5f89b7247b452e6/invenio_app_rdm/theme/templates/invenio_app_rdm/page.html#L10)).

Moderate, but still not blocking. Can merge if opening an issue about it.

Reply:
@ppanero we would need to add a `{{ super() }}` as we override the parent block! But I agree we can fix it in an upcoming PR ;)